tools/devops/automation/scripts/bash/build-nugets.sh

```diff
@@ -18,3 +23,42 @@ DOTNET_PKG_DIR=$(make -C tools/devops print-abspath-variable VARIABLE=DOTNET_PKG
 make -C dotnet package -j
 cp -c "$DOTNET_PKG_DIR"/*.pkg ../package/
 cp -c "$DOTNET_PKG_DIR"/*.msi ../package/
+
+# Down from here we publish mlaunch as .nupkg
```

Comment:
I think this logic could go into its own shell script (say `build-mlaunch-nuget.sh`)